Zapier (#1780)
enkidevs/curriculum: zapier/introduction/what-is-zapier/zapier-dependencies.md

---
author: kapnobatai136

category: must-know

aspects:
  - introduction

---

# Zapier Dependencies

---
## Content

> With so many features and possibilities you'd think that Zapier requires many dependencies or a complicated install process. The truth is, using Zapier is as easy as creating an account.

Reviewer: ```suggestion
With so many features and possibilities, you'd think that Zapier requires many dependencies or a complicated install process. The truth is, you only need an account to get started with Zapier.
```

^ don't mind this, but this paragraph for me sounds like an ad haah

> Your first step is going to [Zapier's official registration website](https://zapier.com/sign-up/) followed by creating an account through either Google, Facebook or just email. That's it, you're now ready to use Zapier.

Reviewer: ```suggestion
To create your account head to the [Zapier's official registration website](https://zapier.com/sign-up/). You can do this through either Google, Facebook or plain email.
```

> The free version of Zapier is very useful for getting used to the tool and creating some personal integrations, but has its limitations.

Reviewer: I'd extend this section a bit more. maybe make a bullet point list with the free features. otherwise this insight is basically "create an account"
